fvm/src/lib.rs

```diff
@@ -134,7 +135,7 @@ mod test {
             Zero::zero(),
             fvm_shared::version::NetworkVersion::V14,
             root,
-            manifest_cid,
+            (0, Some(manifest_cid)),
```

Comment:
Temporary so that we can make a release to integrate in builtin-actors; removed by #404.